build.gradle

```diff
@@ -464,6 +465,26 @@ project('spring-integration-test-support') {
     }
 }
 
+project('spring-integration-debezium') {
+    description = 'Spring Integration Debezium Support'
+    dependencies {
+        api project(':spring-integration-core')
+        api("io.debezium:debezium-api:$debeziumVersion") {
```

Comment:
Cannot we reply on a `debezium-embedded` dep?
Why do we need this extra `debezium-api` one?

We also need to think about excluding those EE deps.
See GH action failure for PR where it shows duplicated classpath entries.

You can run that locally like `./gradlew :spring-integation-debezium checkClasspathForConflicts` commenting out its `onlyIf { isCI }`